scripts/codemod.js

```diff
@@ -1,16 +1,13 @@
 const createProcessor = require('./utils/codemod/createProcessor');
-const { read, write } = require('to-vfile');
+const { write } = require('to-vfile');
+const vfileGlob = require('vfile-glob');
 const codemods = require('../codemods');
 
-const FILE =
-  './src/content/docs/mobile-apps/new-relic-mobile-apps/ios-app/introduction-ios-mobile-app.mdx';
-
-const process = async (path) => {
-  const file = await read(path);
+const process = async (file) => {
   const processor = createProcessor({ codemods });
 
   await processor.process(file);
   await write(file, 'utf-8');
 };
 
-process(FILE);
+vfileGlob('./src/content/**/*.mdx').subscribe((file) => process(file));
```

Comment:
what does `vfileGlob` do?

Reply:
Much like the `read` from `to-vfile` (which we were using before), this allows you to pass a glob pattern and get vfile's for each file in the glob. This allows us to process each file in the glob with the codemod (here being all `.mdx` files inside of `src/content`)

Reply:
https://github.com/shinnn/vfile-glob